assets/agenda/components/AgendaListItem.tsx

```diff
@@ -136,6 +136,15 @@ class AgendaListItem extends React.Component<any, any> {
         // Show headline for adhoc planning items
         const showHeadline = !item.event && get(item, 'headline.length', 0) > 0;
 
+        const descriptionHTML = description.replace('\n', '<br/>').replace('<br/>', '</p>\n<p>');
```

Comment:
not sure this would generate valid html in general, why do we need this?
eg. from:
```html
<ul>
  <li>foo</li>
</ul>
```
you will get:
```html
<ul></p>\n<p><li>foo</li></p>\n<p></ul></p>\n<p>
```

Reply:
we definitely need to talk about this